apps_download/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017-2018 BizzAppDev
+# Copyright 2017 AgentERP
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Product Download for Appstore",
+    "version": "11.0.1.0.0",
+    "author": "BizzAppDev, AgentERP, Elico Corp, "
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/apps-store",
+    "license": "AGPL-3",
+    "category": "Sales",
+    "depends": [
+        'base',
+        'website_sale_digital',
+        'github_connector_odoo',
+        'apps_product_creator',
+    ],
+    "summary": "Product Download for Appstore",
+    "data": [
+        'demo/product_template_demo.xml',
```

Comment:
Shouldn't this be under `demo` key loading this setup only in case of an instance with without_demo=False ?